libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/utils/QueriesProvider.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.navigation.base.internal.utils
+
+import com.mapbox.api.directions.v5.models.DirectionsResponse
+
+/**
+ * Queries params, which add data to [DirectionsResponse] and the SDK cannot mock it
+ * (See [https://docs.mapbox.com/api/navigation/directions](https://docs.mapbox.com/api/navigation/directions)):
+ * - `engine=electric` - electronic vehicle routing. [https://docs.mapbox.com/api/navigation/directions/#electric-vehicle-routing](https://docs.mapbox.com/api/navigation/directions/#electric-vehicle-routing)
+ */
+internal object QueriesProvider {
+    internal val exclusiveQueries: Map<String, String> = mapOf(
```

Comment:
changed to `val` and now invoked directly from `QueriesProvider` because `DirectionsRouteMissingConditionsCheck` initialized via unit tests and stored current values, mock [didn't work](https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/19090/workflows/f1233135-4b2a-46e6-8717-9de5c268adff/jobs/91525)

cc @dzinad